tests/hubspot/test_hubspot_source.py

```diff
@@ -211,14 +262,15 @@ def test_all_resources(destination_name: str) -> None:
             list(row)
             for row in client.execute_sql(
                 """
-                SELECT ch.property_name, ch.value, ch.source_type, ch.source_type, ch.timestamp
+                SELECT ch.property_name, ch.value__v_text, ch.source_type, ch.timestamp
```

Comment:
Not sure what is the `value` field. I tried to extract data (not through tests, but with the pipeline examples, and the value is actually in `value__v_text`).

Reply:
yeah this means that we are getting inconsistent types and `dlt` creates variant columns as value can be integer and then we get string value.

I expect we'll get a request to pivot history table. so we actually generate separate columns for all possible properties ie `email_value). I think we should ignore that for now

Reply:
Well, the tests will not pass with `value`. That's why I edited it.